### ISSUE TITLE
Instant click after typing to select square

### ISSUE THREAD

**Issue by gcerquant** [croian/mouseless-issues#41]
### Problem Description

The added friction to click after selecting a column/row is unnecessary.

### Proposed Solution

I would love to be able to click **_instantly_**:

1. Press `Left-Command` 
2. Type two letters

That's it. No step 3, a click has been made at the center of the square I selected.

### Additional Context

I think I don't care much about scrolling / or be able to select a spot with higher precision.
I prefer to have a VERY FAST solution when not requiring millimeter precision, and reverting back to mouse if needing something really accurate.

I tested a few area, and most of the UI elements I care would have been clickable.

**Comment by vlnn** [comment]
Could it be potentially separate mode (e.g. called with `Right-Command` or other shortcut)? On my retina setup the buttons and checkboxes can be pretty small and default green dot click may be not enough.

**Comment by gcerquant** [comment]
Yes, good idea, a different mode with a different trigger would be perfect.

I know the extra step does not seem much, but I think for me it will be the difference between using it long term or not.

**Comment by croian** [comment]
very interesting use case, i like it.  current config validation rules won't allow it unfortunately, but all the mechanics are in place for this to easily doable.  i'll try to at least loosen the validation rules for this so it can be done in the next release, with a gui update hopefully at the same time or not far after.

some technical info, if you're interested: currently, two-stage cells + the subgrid are hardcoded in as required to validate.  to fix this, I'll simply allow one-stage cell (for choosing the column) and then the subgrid will fill the rows, allowing for an instant click.

**Comment by croian** [comment]
also separate modes is a great idea!  i plan to add allowing for different configs for different monitors not too far in the future, but activating different configs via different hotkeys would be awesome!

**Comment by gcerquant** [comment]
Glad to hear you like it, looking forward to it very much.

Thank you for the tech details, much appreciated

**Comment by croian** [comment]
The implementation of this one is a bit more complex than I initially thought, so haven't gotten to it just yet, sorry, but I plan on prioritizing it after 0.3.0.

**Comment by croian** [comment]
This is done!  Pending release.  Details subject to change but currently implemented by the newly-exposed `strategy` field:

![Image](https://github.com/user-attachments/assets/0b192e60-03ec-41ea-a969-4b1cbcfe61ad)

**Comment by gcerquant** [comment]
Awesome, will check it out. Thank you !

**Comment by alfredomtx** [comment]
@croian is this new `Strategy` option available in the Mac version? I can't find in the UI.

![Image](https://github.com/user-attachments/assets/c586fd73-0160-4dcf-9f5e-09e394dd4d08)

**Comment by bannmann** [comment]
@alfredomtx I think @croian ended up implementing this differently. Do you have the option below?

![Image](https://github.com/user-attachments/assets/574c39bf-9339-432c-a99b-32fed1e3bfcf)

(Disclaimer: I'm on Windows, but my guess is that this is not OS specific.)
